apps/scripts/push_to_docker.sh

```diff
@@ -2,9 +2,16 @@
 
 set -e
 
+IMAGE_NAME=$1
+
+if [ -z "$IMAGE_NAME" ]; then
```

Comment:
nit - I think `set -u` will accomplish the same thing - it will fail the script if $1 is unset. May fail with a less helpful error though so this makes sense too